src/angular/11-declarative-state/declarative-state.md

```diff
@@ -45,193 +45,179 @@ everything about how the state is created. In imperative code you'd need to read
 when all input streams have emitted a value.
 
 @sourceref ./combine.html
-@codepen  
+@codepen
 
 ### Initializing A Stream
+
 A common situation is working with streams that only produce a value after an event, for example when an HTTP request
 completes or when a value changes in a form control. When using a stream like this in your components, you'll likely
-want to have an initial "base state" that your view can use during the initial render. In RxJS this is handled by the 
-[`startWith`](https://rxjs-dev.firebaseapp.com/api/index/operators/startWith) operator, which emits a value when the 
+want to have an initial "base state" that your view can use during the initial render. In RxJS this is handled by the
+[`startWith`](https://rxjs.dev/api/operators/startWith) operator, which emits a value when the
 stream is first subscribed to.
 
 @sourceref ./startWith.html
 @codepen
 
 ### Transforming The Values Of A Stream
-When values are emitted from a stream it's common to transform them in some way before they're used by your application. 
-One operator used for this is the [`map`](https://rxjs-dev.firebaseapp.com/api/index/operators/map) operator, which 
+
+When values are emitted from a stream it's common to transform them in some way before they're used by your application.
+One operator used for this is the [`map`](https://rxjs.dev/api/operators/map) operator, which
 takes an emitted value and returns a modified value that will be passed to the subsequent operators in the stream.
 
 @sourceref ./map.html
 @codepen
- 
+
 ### Emitting Values From Another Stream
-When using a stream you may want to emit values from another stream as part of the original stream. RxJS offers a 
-variety of ways to do this, but the one we'll demonstrate is the 
-[`flatMap`](https://rxjs-dev.firebaseapp.com/api/operators/flatMap) operator. Like the map operator it takes an 
+
+When using a stream you may want to emit values from another stream as part of the original stream. RxJS offers a
+variety of ways to do this, but the one we'll demonstrate is the
+[`mergeMap`](https://rxjs.dev/api/operators/mergeMap) operator. Like the map operator it takes an
 emitted value from a stream, but instead of returning a modified value it returns another stream whose emitted values
 will be passed to the subsequent operators.
 
-@sourceref ./flatMap.html
-@codepen  
+@sourceref ./mergeMap.html
+@codepen
 
-### Handling Multiple Subscribers To A Stream 
-An advanced topic when working with streams is how streams behave when they have multiple subscribers. To understand 
+### Combining Previous And Current Emissions Of A Stream
+
+If you need to compare the current (latest) emitted value with the previous value of a stream, you can use the [`pairwise`](https://rxjs.dev/api/operators/pairwise) operator. Once there are at least 2 emitted values, it starts emitting an array: `[previous, current]`.
```

Comment:
Maybe instead of "array" we could say "tuple"? Or would that just be confusing?

Reply:
I'm fine either way. @tehfedaykin or @vdtdev, any thoughts?

Reply:
gonna resolve for now